### PR TITLE
[ON HOLD] Readme waffle

### PR DESCRIPTION
**Pull request by jverbraeken**
- Relevant Issues: -
- Related Pull Requests: -

* * *

## What
This Pull Request adds to the repository a nice overview of the SCRUM board on the homepage

## Why
This Pull Request is needed because it's nice to have this overview

## How
This feature can be viewed/tested within the project by looking at the Github home page

## Alternative implementation
Other implementations that I've have considered are -

### CODE REVIEW

**Comment by ericcornelissen** [comment]
I agree with @clenrock, I would suggest using the standard one Waffle provides (under Project Settings > General) and add it next to the Travis build status and Coveralls coverage status.

That would be:
[Build Status](https://travis-ci.org/ericcornelissen/NervousFish) [Coverage Status](https://coveralls.io/github/ericcornelissen/NervousFish?branch=master) [Stories in Ready](http://waffle.io/ericcornelissen/nervousfish)

**Comment by jverbraeken** [comment]
Hmm, I'm in doubt. Shall I completely remove this PR or do it the way @ericcornelissen proposed?

**Comment by clenrock** [comment]
@jverbraeken The way @ericcornelissen proposed

**Comment by jverbraeken** [comment]
@clenrock @ericcornelissen Done!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # NervousFish
 [![Build Status](https://travis-ci.org/ericcornelissen/NervousFish.svg?branch=develop)](https://travis-ci.org/ericcornelissen/NervousFish)
 [![Coverage Status](https://coveralls.io/repos/github/ericcornelissen/NervousFish/badge.svg?branch=master)](https://coveralls.io/github/ericcornelissen/NervousFish?branch=master)
+[![Sprint backlog](https://badge.waffle.io/ericcornelissen/nervousfish.png?label=this%20week&title=Sprint%20backlog)](http://waffle.io/ericcornelissen/nervousfish)
```
Comment (ericcornelissen) [inline README.md]: I would prefer the "Ready" column over the "Sprint backlog" column 😇

**Comment by ericcornelissen** [comment]
@jverbraeken, why did you close this PR?

**Pull request closed without merging**